cypress/support/pages/home.pageObject.js

```diff
@@ -7,9 +7,16 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
-  assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+  assertHeaderContainUsername = (username) => {
+    cy.get('[data-cy="username-link"]').should('contain', username);
```

Comment:
You should utilize the previously defined usernameLink method instead of repeating the selector.